ext/date/php_date.c

```diff
@@ -711,6 +711,11 @@ static zend_string *date_format(const char *format, size_t format_len, timelib_t
 
 			/* timezone */
 			case 'I': length = slprintf(buffer, sizeof(buffer), "%d", localtime ? offset->is_dst : 0); break;
+			case 'Q': if (!localtime || strcmp(offset->abbr, "UTC") == 0 || strcmp(offset->abbr, "Z") == 0) {
+					      length = slprintf(buffer, sizeof(buffer), "%s", "Z");
+					      break;
+					  }
+					  /* break intentionally missing */
```

Comment:
I would use lower case `p` instead, and also please indent as is done for `B` above in line 693 to 702.

Reply:
Ok indentation changed and it uses "p" now.